torch/distributed/distributed_c10d.py

```diff
@@ -312,7 +312,8 @@ def init_process_group(backend,
                        world_size=-1,
                        rank=-1,
                        store=None,
-                       group_name=''):
+                       group_name='',
+                       transport=''):
```

Comment:
Why not reuse backend field like gloo-tls instead of adding a new param transport.

Reply:
I see your point. Basically we want to keep the interface as it is. Will address

Reply:
I see your point. Basically we want to keep the interface as it is. Available backends are hardcoded in torch/distributed/distributed_c10d.py as below. Adding GLOO-TLS may be tricky to isolate from OSS

  60     UNDEFINED = "undefined"
  61     GLOO = "gloo"
  62     NCCL = "nccl"
  63     MPI = "mpi"
  64     TCP = "tcp"